net-dns/dnscrypt-proxy/files/dnscrypt-proxy.initd-2

```diff
@@ -0,0 +1,11 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
2018

Reply:
@gyakovlev I propose you to close this PR and switch on your branch.
I will comment your sources.